.github/workflows/der.yml

```diff
@@ -61,13 +66,14 @@ jobs:
             platform: ubuntu-latest
             rust: stable
 
+          # temporary disable, since cargo-hack installation does not work yet
           # 64-bit Windows
-          - target: x86_64-pc-windows-msvc
-            platform: windows-latest
-            rust: 1.57.0 # MSRV
-          - target: x86_64-pc-windows-msvc
-            platform: windows-latest
-            rust: stable
+          #- target: x86_64-pc-windows-msvc
+          #  platform: windows-latest
+          #  rust: 1.57.0 # MSRV
```

Comment:
Windows tests are disabled for now since the `cargo-hack-install` action does not currently work on Windows. It fails with `wget: command not found`, IIUC `wget` should be installed by default, but, unfortunately, we have to use full path to it.

AFAIK `der` is platform independent, so disabling those tests should not be a big issue.